### ISSUE TITLE
Failed to find minhash filter in kibana Dev app

### ISSUE THREAD

**Issue by Fred12** [codelibs/elasticsearch-minhash#17]
Hello,
after installing minhash filter in elasticsearch, I started kibana app and in the Dev tool, I copy&pasted the code to create the minhash filter mapping: 
curl -XPUT 'localhost:9200/my_index' -d '{
  "index":{
    "analysis":{
      "analyzer":{
        "minhash_analyzer":{
          "type":"custom",
          "tokenizer":"standard",
          "filter":["minhash"]
        }
      }
    }
  }
}' 

But then, I got this error:

{
  "error": {
    "root_cause": [
      {
        "type": "illegal_argument_exception",
        "reason": "Custom Analyzer [minhash_analyzer] failed to find filter under name [minhash]"
      }
    ],
    "type": "illegal_argument_exception",
    "reason": "Custom Analyzer [minhash_analyzer] failed to find filter under name [minhash]"
  },
  "status": 400
}

**Comment by marevol** [comment]
I think the plugin installation is failed.

**Comment by Fred12** [comment]
nope, the weirdo tutorial on this github page does not work, I used the code from this thread:
https://github.com/codelibs/elasticsearch-minhash/issues/15

and got it to work, I am just trying to figure out how I can do a search query, or a "more_like_this" query on the min_hash field.. it just doesn't work because its not text or term and I cannot convert it to text/ term.. how would I normally do it?
